plugins/inputs/varnish/varnish_test.go

```diff
@@ -114,6 +125,12 @@ MEMPOOL.vbc.pool                   10          .   In Pool
 MEMPOOL.vbc.sz_wanted              88          .   Size requested
 `
 
+var admOutput = `
+available   auto/warm          0 boot
```

Comment:
What version of varnish-cache are you using? For me the output from varnishadm looks like this:

`active      auto warm          0 reload_20200301_181101_4812`

Reply:
Hi @helenosheaa ,

If it can help you, on Varnish 6.5:
- After a restart:
active   auto    warm         0    boot
- After a reload:
active      auto    warm         2    reload8

Reply:
But can be interesting to extract only the backend name from VBE instead to take into account the reload name
Example via varnishstat:
Via boot:
Replace the name "VBE.boot.node1.happy" by "VBE.node1.happy" 
Via reload:
Replace the name "VBE.reload8.node1.happy" by "VBE.node1.happy"